compiler/compile/OMRCompilation.cpp

```diff
@@ -2730,3 +2730,12 @@ void OMR::Compilation::invalidateAliasRegion()
       new (&self()->_aliasRegion) TR::Region(_heapMemoryRegion);
       }
    }
+
+// This query is for temporarily disable opts not supporting
+// the newly added rdbar/wrtbar opcodes. Subprojects can overrite
+// the answer. This query should be deleted eventually if
+// all the optimizations support the opcodes.
```

Comment:
Can you use doxygen formatting for this comment block?

Reply:
Changed to use doxygen formatting.